.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: go
 go:
-  #- 1.5  go test arguments are different in 1.5 :-(
-  - 1.6
```

Comment:
Are we broken on 1.6?  The challenge is that many people are still running with go 1.6.  I'm not sure that we build kops itself in a docker container, so we want to keep testing against 1.6 if we can.

In any case there should be a comment about why we're not testing against it, and we shouldn't remove the 1.5 comment.